MediaBrowser.Controller/MediaEncoding/EncodingHelper.cs

```diff
@@ -2329,22 +2329,26 @@ private string GetFixedSizeScalingFilter(Video3DFormat? threedFormat, int reques
         /// <summary>
         /// Gets the number of threads.
         /// </summary>
-        public int GetNumberOfThreads(EncodingJobInfo state, EncodingOptions encodingOptions, string outputVideoCodec)
+        public static int GetNumberOfThreads(EncodingJobInfo state, EncodingOptions encodingOptions, string outputVideoCodec)
```

Comment:
if state can be null (line 2342), andd outputVideoCodec is passed null later in the code, shouldn't it be

```
public static int GetNumberOfThreads(EncodingJobInfo? state, EncodingOptions encodingOptions, string? outputVideoCodec)
```